### DISCUSSION TITLE
Volunteer in Translation - Alice

### DISCUSSION THREAD

**Discussion by Guedesou** [TheAliceProject/alice3#257] [Q&A]
### **Hello, people!** 
**I wanna ask if you need help in translate Alice in Portuguese (BR).** 
I already helped a lot of cryptocurrencies in translations to get some coins, but I love working as Volunteer. 
So... if you wanna help, please let me know!
_PS: I made my account on Zanata.
![gg](https://user-images.githubusercontent.com/56264781/168621952-583b2b09-dba7-4466-85c4-9ccaf2da064c.PNG)
 ;)_

**Comment by MelanieLam** [comment]
If you are still interested in translation: https://github.com/TheAliceProject/alice3/wiki/Contributing-to-Translation-Efforts

**Reply by Guedesou** [reply]
Of course! Lets make it. 🖤🙌🏻
